scripts/eurostat/regional_statistics_by_nuts/README.md

```diff
@@ -0,0 +1,104 @@
+# Importing EuroStat Regional Statistics by NUTS Into Data Commons
+
+> This README is a template of what an import's README might look like.
+
+Author: eftekhari-mhs
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+1. [Import Artifacts](#import-artifacts)
+1. [Generating Artifacts](#generating-artifacts)
```

Comment:
These are broken links (subsections don't exist?

Reply:
yes, looks like we have one section.